skrub/_scaling_factor.py

```diff
@@ -0,0 +1,106 @@
+import numpy as np
```

Comment:
It's always good to have a top-level docstring for a file that describes briefly what the file does. It helps people reading the codebase